### PR TITLE
Intial configuration

### PR DESCRIPTION
**Pull request by msotoperdomo**
Initial configuration for the base image for php applications.
- Include Consul template 
- Include base php libraries
- Php.ini and php-fpm.conf defined as consul keys
- Base libraries use by applications
- Documentation

### CODE REVIEW

File: bin/functions [hunk 1]
```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
```
Comment (nhuray) [inline bin/functions:1]: This file shouldn't be commited ! You have to add it to the `.gitignore`

File: rootfs/usr/local/bin/reload-config.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+echo "Reload script"
+pid_php=$(cat /usr/local/var/run/php-fpm.pid)
+if [ -n "$pid_php" ]; then
+  echo "Reloading php configuration"
+  kill -USR2 $pid_php
```
Comment (nhuray) [inline rootfs/usr/local/bin/reload-config.sh]: Cool stuff !

File: versions/1.0.0/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+RUN apk update && apk upgrade \
+    && apk add --update \
+      coreutils \
+      freetype-dev \
+      libjpeg-turbo-dev \
+      libltdl \
+      libmcrypt-dev \
+      libpng-dev \
+      curl wget bash tree jq bind-tools su-exec \
+    && apk add --no-cache --update libmemcached-libs zlib \
```
Comment (nhuray) [inline versions/1.0.0/Dockerfile]: Why do you need to run those commands ? They are not defined in your base image ?

File: bin/docker-php [hunk 4]
```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env bash
```
Comment (nhuray) [inline bin/docker-php]: For the other Docker images, we named the script based on the name of the image, so `php` instead of `docker-php` !

**Review by nhuray — CHANGES_REQUESTED** [review]
Some changes ! Great job 👍

File: rootfs/usr/local/bin/reload-config.sh [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+echo "Reload script"
+pid_php=$(cat /usr/local/var/run/php-fpm.pid)
+if [ -n "$pid_php" ]; then
+  echo "Reloading php configuration"
```
Comment (bratislavml) [inline rootfs/usr/local/bin/reload-config.sh]: 👍

**Review by bratislavml — APPROVED** [review]
👍 OK nothing to add beside @nhuray

**Pull request merged**